### PR TITLE
feat: add katana v3 dex ronin

### PR DESCRIPTION
**Pull request by cxheng315**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

1. Rename original katana dex to show v2
2. Add ronin v3 dex support 

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by cxheng315** [comment]
@Hosuke can you check this please 
`Model 'model.dex.katana_v2_ronin_base_trades' (models/trades/ronin/platforms/katana_v2_ronin_base_trades.sql) depends on a source named 'katana_dex_ronin.KatanaPair_evt_Swap' which was not found`
but I did get it in Dune
```sql
SELECT * FROM katana_dex_ronin.KatanaPair_evt_Swap
LIMIT 1
```

File: sources/_sector/dex/trades/ronin/_sources.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 version: 2
 
 sources:
-  - name: katana_dex_ronin
+  - name: katana_v2_dex_ronin
     tables:
       - name: KatanaPair_evt_Swap
-      - name: KatanaFactory_evt_PairCreated
\ No newline at end of file
+      - name: KatanaFactory_evt_PairCreated
```
Comment (Hosuke) [inline sources/_sector/dex/trades/ronin/_sources.yml:7]: We need to make sure the source here match the name in dune.

Comment (cxheng315) [reply sources/_sector/dex/trades/ronin/_sources.yml]: Thank you so much!
Should be solved now!

File: dbt_subprojects/dex/seeds/trades/katana_v3_ronin_base_trades_seed.csv
Comment (Hosuke) [inline dbt_subprojects/dex/seeds/trades/katana_v3_ronin_base_trades_seed.csv:1]: We can merge contents into the original seed file.

File: dbt_subprojects/dex/models/trades/ronin/_schema.yml [hunk 2]
```diff
@@ -5,19 +5,36 @@ models:
...
     data_tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - tx_hash
             - evt_index
       - check_dex_base_trades_seed:
-          seed_file: ref('katana_ronin_base_trades_seed')
\ No newline at end of file
+          seed_file: ref('katana_v2_ronin_base_trades_seed')
+
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/ronin/_schema.yml:24]: And add filter here.

File: dbt_subprojects/dex/models/trades/ronin/_schema.yml [hunk 3]
```diff
@@ -5,19 +5,36 @@ models:
...
+    config:
+      tags: ["ronin", "dex", "trades", "katana"]
+    description: "Katana v3 ronin base trades"
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('katana_v3_ronin_base_trades_seed')
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/ronin/_schema.yml:40]: And here.
```suggestion
      - check_dex_base_trades_seed:
          seed_file: ref('katana_ronin_base_trades_seed')
          filter:
            version: 3
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: dbt_subprojects/dex/seeds/trades/katana_v3_ronin_base_trades_seed.csv
Comment (cxheng315) [reply dbt_subprojects/dex/seeds/trades/katana_v3_ronin_base_trades_seed.csv:1]: thank you, just pushed the commit 🙏

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @cxheng315 

[Sample katana_v2_ronin_base_trades](https://dune.com/queries/4457509)
[Sample katana_v3_ronin_base_trades](https://dune.com/queries/4457514)

**Pull request merged**